### PR TITLE
Move C+L at NodeConfCo to the past

### PR DESCRIPTION
**Pull request by julianduque**
Event was a success!! now it's time to move it to the past ;)

### CODE REVIEW

**Review by Trott — APPROVED** [review]

**Comment by ghost** [comment]
@nodejs/website：Please check your own language so as to confirm that 'To be announced' and other names of events are correctly translated, I used English in union.

**Comment by e-jigsaw** [comment]
ja translated :)

**Review by edsadr — APPROVED** [review]

File: locale/ar/get-involved/code-and-learn.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ layout: contribute.hbs
 
 ## فعاليات برمج + تعلم قادمة
 
-- ميدلين ، كولومبيا في 21 و 22 جوان [NodeConfCo](https://colombia.nodeconf.com/)
+- To be accounced
```
Comment (Aissaoui-Ahmed) [inline locale/ar/get-involved/code-and-learn.md]: To be accounced > سوف يتم الإعلان عنها

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

**Review by aymen94 — APPROVED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Comment by Trott** [comment]
Should this land (to update the listing on the website) and any additional translations can happen later?

**Comment by bnb** [comment]
@Trott I would assume that is the process 🤔

**Pull request merged**